src/modules/components/proposal/proposalVoting/proposalVotingBodyContent/proposalVotingBodyContent.tsx

```diff
@@ -0,0 +1,67 @@
+import classNames from 'classnames';
+import { useEffect, useState, type ComponentProps } from 'react';
+import { Button, IconType } from '../../../../../core';
+import { useGukModulesContext } from '../../../gukModulesProvider';
+import { ProposalVotingStatus } from '../../proposalUtils';
+import { ProposalVotingTab } from '../proposalVotingDefinitions';
+import { useProposalVotingStageContext } from '../proposalVotingStageContext';
+import { ProposalVotingTabs } from '../proposalVotingTabs';
+
+export interface IProposalVotingBodyContentProps extends ComponentProps<'div'> {
+    /**
+     * Status of the stage.
+     */
+    status: ProposalVotingStatus;
+    /**
+     * Name of the proposal stage displayed for multi-stage proposals.
+     */
+    name?: string;
+    /**
+     * plugin address of the body used to determine if the content should be rendered or not.
+     */
+    bodyId?: string;
+}
+
+export const ProposalVotingBodyContent: React.FC<IProposalVotingBodyContentProps> = (props) => {
+    const { bodyId, children, name, status, ...otherProps } = props;
+    const { className } = otherProps;
```

Comment:
A custom className would override the existing classes otherwise
```suggestion
    const { bodyId, children, name, status, className, ...otherProps } = props;
```